app/controllers/candidate_interface/sign_up_controller.rb

```diff
@@ -16,13 +16,13 @@ def create
       @sign_up_form = CandidateInterface::SignUpForm.new(candidate_sign_up_form_params)
 
       if @sign_up_form.existing_candidate?
-        MagicLinkSignIn.call(candidate: @sign_up_form.candidate)
+        CandidateInterface::RequestMagicLink.call(candidate: @sign_up_form.candidate)
```

Comment:
The `call` convention makes this and `RequestMagicLinkSignup` a bit fiddly to read — maybe have an interface sth like this, and the namespace is bit more cohesive?

```
CandidateInterface::MagicLinks.send_sign_in_link
CandidateInterface::MagicLinks.send_sign_up_link
```

Reply:
Agree 👍

Perhaps this?

```
CandidateInterface::RequestMagicLink.for_sign_in
CandidateInterface::RequestMagicLink.for_sign_up
```

Reply:
Seems good to me. Have updated.